src/main/java/com/restaurantpicker/RestaurantpickerController.java

```diff
@@ -6,12 +6,15 @@
 @Controller
 public class RestaurantpickerController 
 {
+

```

Comment:
Added comment since once more request mappings are added it'll be difficult to keep track of what does what.